Set/Models/GameModel.swift

```diff
@@ -122,20 +148,23 @@ struct GameModel {
         }
     }
     
-    mutating private func getBonus(for player: Player) {
```

Comment:
here you had hardcoded`Constants.defaultBonus` as a value for increasing value, just change name of function to match it's mutation behaviour, and then just add another one parameter that will tell function amount on which it should increase score